samples/javascript_nodejs/50.teams-messaging-extensions-search/bots/teamsMessagingExtensionsSearchBot.js

```diff
@@ -3,30 +3,113 @@
 
 const axios = require('axios');
 const querystring = require('querystring');
-const { TeamsActivityHandler, CardFactory } = require('botbuilder');
+const { TeamsActivityHandler, CardFactory,MessageFactory, ConsoleTranscriptLogger } = require('botbuilder');
+var fs = require('fs');
+const AdaptiveCard = require('../Resources/RestaurantCard.json');
+const ConnectorCard = require('../Resources/ConnectorCard.json');
+var configuration = require('dotenv').config();
+var env = configuration.parsed;
+var baseurl = env.BaseUrl;
+var publicDir = require('path').join(__dirname,'../public/Images'); 
```

Comment:
Sorry, looks like one of my comments didn't save in my last review round. Please replace all instances of `var` with either `const` or `let`. There's more than just this section, too.